imaging/ml/ml_codelab/README.md

```diff
@@ -1,14 +1,14 @@
-# ML Codelabs on Cloud Healthcare API
+#  AI Platform Notebooks on Cloud Healthcare API
```

Comment:
Remove extra space in the beginning.
I am not sure about changing "ML Codelabs" to "AI Platform Notebooks"... "AI Platform Notebooks" is just a service that we are now using instead of Cloud Datalab to run these notebooks. I suggest changing to "Imaging ML Codelabs". 

I am not  "ML Codelabs" is more specific than "AI Platform Notebooks
These are still codelabs. We were recommending to run them

Reply:
done